.github/workflows/project-actions.yml

```diff
@@ -8,6 +8,9 @@ on:
     types:
       - opened
       - reopened
+    branches:
+      - main
+      - dev
```

Comment:
Sorry I also think this shouldn't be in here, will cause a merge conflict when goes back to `main`

Reply:
done